README.md

```diff
@@ -32,12 +32,44 @@ docker run --rm -p 27017:27017 -e MONGO_INITDB_ROOT_USERNAME=root -e MONGO_INITD
 Run `docker compose up` and navigate to `http://localhost:3000/` in your browser.
 
 ### Running Testcase in Local
+#### For Rspec tests, run
+```
+bundle exec rspec
+```
 
-For Jest and Cucumber tests, run
+#### For Jest tests, 
+1. Setup the application locally (without docker compose)
+2. Run
 ```
- yarn test --coverage
+ yarn coverage spec/javascript
 ```
-For Rspec tests, run
+
+#### For Cucumber tests,
+1. Setup the application locally (without docker compose)
+2. Create an admin with email: `admin@example.com` and password: `123456qt`
+3. Create an event:
+
+| Event title | Event description | Date | State | Location | Category | Paid |
+|--------|--------|--------|--------|--------|--------|--------|
+| Miu Miu | Miu Miu showcase  | 2024-12-19 | New York | Albany   | Fashion  | Yes  |
+4. Run 
 ```
-bundle exec rspec
-```
\ No newline at end of file
+yarn test spec/features
+```
+5. If you face timeout errors, run `rails webpacker:install && rails webpacker:compile` before running this test.
+
+### Deployment
+This application uses the following components:
+
+Backend
+* Ruby v2.6
+* Rails v5.2
+
+Frontend
+* React v17.0
+* Material UI v4.12
+
+Database
+* MongoDB
+
```

Comment:
In Heroku,
1. Create an app <app-name>
2. Add MONGODB_URI in the app.
3. Edit the deploy.yaml file to use <app-name>